src/stories/Blocks/autosuggest/Autosuggest.stories.tsx

```diff
@@ -25,42 +25,9 @@ export default {
 const Template: ComponentStory<typeof Autosuggest> = (
   args: AutosuggestProps
 ) => (
-  // Apart from <Autosuggest />, everything else is here just for the story
-  // context. The autosuggest styling is directly dependent on the header component,
-  // Please keep up to date with Header.tsx
-  <header className="header" style={{ height: "144px" }}>
```

Comment:
Is it necessary to remove this? As I see it, autosugest needs to be in the header to give the right impression

Reply:
As I write in the commit:

> 
> Update autosuggest story to not be dependent on header.
> It's not maintainable nor necessary for us to display the whole
> header in the Autosuggest story.
> If it is necessary in the future, we'll have to make the header
> element a proper element, that can take the auto-suggest element.